tests/baselines/reference/contextualTypeAny.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/contextualTypeAny.ts(3,5): error TS2322: Type '{ p: string; q: any; }' is not assignable to type '{ [s: string]: number; }'.
+  Property 'p' is incompatible with index signature.

```

Comment:
It's not clear what the type of `p` is without the full type, so it'd be better to just report the index signature given that it'd take relatively little effort on our part.